trulens_eval/examples/expositional/models/hugging__Face_.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>It'd be useful here to show usage of this evaluator as part of a recorded app, e.g. as shown in <a href="https://www.trulens.org/trulens_eval/langchain_quickstart" target="_blank">https://www.trulens.org/trulens_eval/langchain_quickstart/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/950/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>